docs/https.md

```diff
@@ -0,0 +1,35 @@
+---
+id: https
+title: HTTPS
+sidebar_label: HTTPS
+---
+
+## HTTPS/TLS
+
+In order to enable HTTPS, the `HTTPS_CERTFILE` and `HTTPS_KEYFILE` environment variables must be set. During development, this may be set to the self-signed certificate that comes with the repository:
+
+```bash
+$ export HTTPS_CERTFILE=cert/selfsigned.pem
+$ export HTTPS_KEYFILE=cert/selfsigned_key.pem
+$ mix phx.server
+```
+
+WARNING: only use the generated certificate for testing in a closed network environment, such as running a development RIG instance on localhost. For production, staging, or testing on the public internet, obtain a proper certificate, for example from [Let’s Encrypt](https://letsencrypt.org/).
+
+For production you should use proper HTTPS certificates instead (for that reason the Docker image comes without certificates).
+
+1. Create your own not-selfsigned certificate
```

Comment:
```suggestion
1. Create your own certificate
```

Depending on the situation, a self-signed certificate might actually _okay_. The important thing here is that no-one uses the one from the repo :)